### DISCUSSION TITLE
Install on Debian

### DISCUSSION THREAD

**Discussion by ghost** [wintercms/winter#84] [Q&A] [answered]
Hello dear Team,
Could it be possible to install the same as way below and get an install-master folder ? Currently empty 
wget https://codeload.github.com/wintercms/install/zip/master -O wintercms.zip

Wish you all the best to come

Best regards

**Comment by bachvtuan** [comment]
You can try install by download release version. Here is latest https://github.com/wintercms/winter/archive/refs/tags/v1.1.2.zip

**Comment by ghost** [comment]
Hello, thanks for this reply,
I'm feeling sorry because I'm a non-technical user, 
this is the way I used to install in the past with an install.php file inside the folder, similar to the one below, I'm hoping it could be done soon.
https://codeload.github.com/octobercms/install/zip/master

Cordially

**Comment by bennothommo** [comment, marked as answer]
@steeve-legal We are currently in the process of rebranding the installer. Once this is done, you will be able to install Winter CMS using the Wizard installer (which is the Zip file you are talking about).

**Reply by bennothommo** [reply]
As [mentioned by](https://github.com/wintercms/winter/discussions/84#discussioncomment-1250987) @arvislacis, the Web Installer is now available, and is the best way for *non-technical* users to install Winter CMS. We also are now part of the [Softaculous library](https://www.softaculous.com/softaculous/apps/cms/WinterCMS), which provides a simple way of installing Winter CMS on shared hosting.

**Comment by ghost** [comment]
Hello @bennothommo , thank you very much, I'm looking forward about this good new and will stay tuned.

Best Regards

**Comment by arvislacis** [comment]
Web installer of Winter CMS has been available for some time now: https://github.com/wintercms/web-installer
